cdap-common/bin/common.sh

```diff
@@ -232,6 +232,11 @@ set_hive_classpath() {
 
       if [[ $(which hive 2>/dev/null) ]]; then
         HIVE_VAR_OUT=$(hive -e 'set -v' 2>/dev/null)
+        __ret=$?
+        if [ ${__ret} -ne 0 ]; then
+          echo "ERROR - Problem running: hive -e 'set -v'"
```

Comment:
intention is to give context around why we are running the command to begin with, not why it failed.